specification/cdn/resource-manager/Microsoft.Cdn/stable/2020-09-01/afdx.json

```diff
@@ -5491,32 +5491,128 @@
       }
     },
     "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/profiles/{profileName}/getLogAnalyticsMetrics": {
```

Comment:
The URL should not have get. Has this API been reviewd?

Reply:
No, this was not reviewed by the ARM team. The engineer on the China team gave me all the log analytics ones so I just pasted them here. We can ask her to schedule a review if needed.